Kumi.Game/KumiGameBase.cs

```diff
@@ -87,13 +81,17 @@ private void load()
             RelativeSizeAxes = Axes.Both,
             Child = new DrawSizePreservingFillContainer() // TODO: Add a way to change the resolution and UI scale dynamically.
             {
-                TargetDrawSize = new Vector2(1920, 1080),
+                TargetDrawSize = new Vector2(1280, 720),
```

Comment:
i don't believe using a resolution lower than FHD is suitable for most modern displays anymore; especially since there's a lack of ui scaling

before moving the base back to 720p, i'd reccomend actually implementing that feature beforehand

Reply:
epic